front/app/services/baskets.ts

```diff
@@ -49,7 +51,14 @@ export function basketByIdStream(
 
 export async function addBasket(object: INewBasket) {
   const basket = await streams.add<IBasket>(apiEndpoint, { basket: object });
-  await streams.fetchAllWith({ dataId: [object.participation_context_id] });
+
+  if (object.participation_context_type === 'Project') {
```

Comment:
We should make a note of this for the phases PR.

Reply:
When converting endpoints I think it's good practice to global search for `fetchAllWith` and convert it where necessary, that way you don't miss anything by accident